diff --git a/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js b/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js
index 17154bbc92ed94200b9977f3ed22558124a79ac6..ccb556690e10148c7320f93b09ff9f8eaa6f8204 100644
--- a/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js
+++ b/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js
@@ -782,7 +782,7 @@ export default class MediaHandler {
     }
     blob.uri = this._relativeMediaUri
       ? `./media_${hash}.${ext}${fragment}`
-      : `https://${ref}--${repo}--${owner}.aem.page/media_${hash}.${ext}${fragment}`;
+      : 'https://' + ref + '--' + repo + '--' + owner + '.' + (process.env.HLX_PROD_SERVER_HOST_PAGE || 'aem.page') + '/media_' + hash + '.' + ext + fragment;
     return blob;
   }
 
