diff --git a/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js b/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js
index d15e299..2a7f892 100644
--- a/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js
+++ b/node_modules/@adobe/helix-mediahandler/src/MediaHandler.js
@@ -63,13 +63,13 @@ export default class MediaHandler {
     }
 
     Object.assign(this, {
-      _awsRegion: opts.awsRegion = 'us-east-1',
+      _awsRegion: opts.awsRegion || process.env.AWS_REGION || 'us-east-1',
       _awsAccessKeyId: opts.awsAccessKeyId,
       _awsSecretAccessKey: opts.awsSecretAccessKey,
       _r2AccountId: opts.r2AccountId,
       _r2AccessKeyId: opts.r2AccessKeyId,
       _r2SecretAccessKey: opts.r2SecretAccessKey,
-      _bucketId: opts.bucketId || 'helix-media-bus',
+      _bucketId: opts.bucketId || 'helix-media-bus-' + (process.env.HELIX_BUCKET_SUFFIX || ''),
       _contentBusId: contentBusId,
       _owner: owner,
       _repo: repo,
